### ISSUE TITLE
Delete Maxar Standard as imagery option

### ISSUE THREAD

**Issue by adiatmad** [hotosm/tasking-manager#5317]
Maxar Standard is no longer available for us (https://josm.openstreetmap.de/ticket/21839). Is it necessary to remove it from the list of options in the Instruction? So, the project creator/manager will choose Maxar Premium, Bing, or other imageries as alternatives for the next project.
![image](https://user-images.githubusercontent.com/38268616/185894892-9969117b-b479-4639-afc8-9e6e8784cc2d.png)

**Comment by SColchester** [comment]
Yes this would be great to remove, it is confusing to Project creators to see "Maxar Standard" as an option - when it is not really an option.
